packages/component-submission/server/resolvers/uploadSupportingFile.js

```diff
@@ -1,14 +1,15 @@
+const config = require('config')
 const User = require('@elifesciences/component-model-user').model
-const Manuscript = require('@elifesciences/component-model-manuscript').model
 const { S3Storage } = require('@elifesciences/component-service-s3')
-
-const { SupportingFiles } = require('../use-cases')
+const { SupportingFiles, Manuscript } = require('../use-cases')
 
 async function uploadSupportingFile(_, { file, id }, { user }) {
   const userUuid = await User.getUuidForProfile(user)
   const files = new SupportingFiles(S3Storage, id, userUuid)
   await files.upload(file)
-  const manuscript = await Manuscript.find(id, userUuid)
-  return manuscript
+
+  const manuscript = new Manuscript(config, userUuid, S3Storage)
```

Comment:
I think this needs to be await `Manuscript.find(id, userUuid)` ?

Reply:
I believe the `manuscript.find` happens inside `getView`

Reply:
that's correct, its now the use case object which handles that part